Project 1/styles.css

```diff
@@ -0,0 +1,56 @@
+body {
+
+    padding-left: 120px;
+    padding-right: 120px;
+    max-width: 700px;
+    min-width: 500px;
+    background-color: rgb(113, 17, 17);
+    margin: 0 auto;
+}
+
+header {
+    background-color: #f7f7f7;
+    max-width: 300px;
```

Comment:
So, an interesting thing is happening here, which you may not have spotted:
<img width="1138" alt="Screen Shot 2019-09-29 at 4 27 57 PM" src="https://user-images.githubusercontent.com/2653980/65838946-7a0c0e80-e2d6-11e9-9da3-6efe11f04fc4.png">

You'll see in this image that your `header` is 370 px wide -- but you have a `max-width` of 300 px! What gives?

The browser is still using the default `box-sizing` -- if you add a `box-sizing: border-box;` rule to your stylesheet, you'll be able to more accurately describe the `max-width` you truly want and not have this head scratcher outcome :)